sidebars.js

```diff
@@ -95,6 +95,7 @@ module.exports = {
       collapsed: false,
       items: [
         "go/introduction",
+        "go/getting-started",
```

Comment:
should this live under Tutorials instead?

Reply:
I suppose for now it could live under either - I am envisioning this whole category will shift in its presentation to something like this:
Go -> Go developer guide
- Getting Started
- Scheduling
- Cancelling
- Signals & Queries
- etc...

And then maybe have a whole different category for Go developer tutorials. It will become clearer I think, as we move things to be reusable pages like this.